R/workflow.R

```diff
@@ -425,6 +431,40 @@ copy_nonmem_model_from <- function(
 }
 
 
+#' Copy a NONMEM control stream file
+#'
+#' Helper function to copy a NONMEM control stream file and optionally update the description, etc. in the new file.
+#' Note that any file existing at `.new_model_path` will be overwritten.
+#' @param .parent_model_path Path to the control stream to copy
+#' @param .new_model_path Path to copy the new control stream to
+#' @param .update_model_file Boolean for whether to update the `$PROBLEM` line in the new control stream. By default it is TRUE, but if FALSE is passed `{.new_model}.[mod|ctl]` will be an exact copy of its parent control stream.
+#' @param .description Description of new model run. This will be passed into the `$PROBLEM` of the new control stream (if `.update_model_file=TRUE`).
```

Comment:
don't forget to export

Reply:
I was actually thinking to keep this one private. I feel like we should encourage people to just use `copy_model_from()` which will call this internally. Can you think of a reason people would want this as a standalone?